.scalafix.conf

```diff
@@ -1,21 +1,9 @@
 rules = [
   RemoveUnused,
-  DisableSyntax,
```

Comment:
Why are we removing this? Those lints look useful. No XML literals in this code base, please.

Reply:
We can keep it, but there seems to be overlap with wartremover and scalafmt.. I had to disable scalafix for certain checks, in places where we had already disable the same check for wartremover, the syntax for disabling was clashing between the tools, etc etc

Reply:
Let's keep it for now, unless it gets totally crazy. If I had to choose between the two it's Wartremover that would be removed.